recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   md5: d04fba2d9245e661f245de0577f48a33
 
 build:
-  number: 11
+  number: 1011
```

Comment:
I guess this is correct? Or should it be 1000?

Reply:
It is always +1000, so it is OK.